packages/components/src/components/hds/app-header/index.hbs

```diff
@@ -0,0 +1,34 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<div class={{this.classNames}} ...attributes>
+  {{!-- GlobalItems --}}
+  <div class="hds-app-header__sub-section">
+    {{! Intended for: ProductMenu, HomeLink/Logo }}
+    {{yield to="globalItemsBefore"}}
+    {{! Intended for: OrgPicker }}
+    <div class="hds-app-header__global-items-mobile-view">
+      {{yield to="globalItemsAfter"}}
+    </div>
+  </div>
```

Comment:
Refactored to divide `:globalItems` into `:globalItemsBefore` & `:globalItemsAfter`. This is to allow the globalItemsAfter content to be rendered within the responsive dropdown menu which is rendered in mobile/tablet view.

(Not sure if this is the best naming or best way to implement this.)

Reply:
It's difficult to know what to call the blocks since we don't restrict what's contained in them.

The `:globalItemsBefore` contains only the HomeLink/Logo right now but in phase 2 it could potentially contain the Product menu before the logo (although there was talk that it would maybe be dropped.)

The `:globalItemsAfter` contains only the Org switcher right now but it's not restricted to it. It doesn't seem like it would be used for anything else from the design explorations but I don't know for sure.

Here's a screen shot of the potential content for the "global items" (from an older design):
<img width="577" alt="image" src="https://github.com/user-attachments/assets/badadde6-b760-4b7e-9e8e-43c8fd7de867">

Reply:
I could rename the first block (:globalItemsBefore) to `:logo` (but then what do I do if phase 2 includes the Product menu, add another named block?)

Then I could call the 2nd block (:globalItemsAfter) just `:globalItems`.

@jorytindall do you have any ideas on naming?

Reply:
Hmmmmm, thinking about this a little bit, within the `global` block there are almost primary and secondary elements within that hierarchy. Maybe for now it should simply be `:logo`, I'm not sure about the horizon of the `AppSwitcher`, but it somewhat seems like that idea has been abandoned. Additionally, that design was also somewhat speculative, I'm not sure if we would go the same direction if we needed to implement something like it in the future.

Then the next block could be `:globalItems`, that makes sense to me.

Reply:
Thanks for weighing in. Ok, I think I will go ahead with renaming those blocks to `:logo` & `:globalItems` as that seems to make the most sense.